packages/lab/src/layout/LayerLayout/LayerLayout.css

```diff
@@ -0,0 +1,112 @@
+.uitkLayerLayout {
+  --layer-border-color: var(--uitkLayerLayout-border-color, var(--uitk-container-border-color-medium));
+  --layer-box-shadow: var(--uitkLayerLayout-box-shadow, var(--uitk-overlayable-shadow-modal));
+  --layer-background-color: var(--uitkLayerLayout-background-color, var(--uitk-container-background-medium));
```

Comment:
try to leave out the -color in -background tokens and just use the background attribute too, which will allow people to use images/gradients etc if they wish